ipstack/plugin.spec.yaml

```diff
@@ -25,6 +25,7 @@ connection:
     title: API Token
     description: API Token
     required: true
+    example: e6541d-7f855e-eac9a1478afeb48fcbc4
```

Comment:
Use this example (this is the format for the key): 349037a927fc2abfd850349c3acd95cdb

Reply:
I can see this being is issue later on, when a user is expecting to see `-` in the APIKey for certain services but sees an example that doesn't have those hypins. Making them wonder if they have the right APIKey